sdk/python/feast/job_service.py

```diff
@@ -0,0 +1,67 @@
+from concurrent.futures import ThreadPoolExecutor
+
+import grpc
+
+import feast
+from feast.core import JobService_pb2_grpc
+
+
+class JobServiceServicer(JobService_pb2_grpc.JobServiceServicer):
```

Comment:
I recommend we follow pep8 style guide as stated [here](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names), to utilize lowercase separated by underscores for function names.

Reply:
This is autogenerated code by grpc, I don't think we have a control to modify it.